sphinx/source/work_in_progress/index.rst

```diff
@@ -0,0 +1,9 @@
+Work in progress
+================
+
+Diagrams that represent the work in progress on the ePBFT/CCF integration
```

Comment:
Let's write a note or warning here to say that operators, users, members and developers of apps should ignore this page.